component-samples/demo/scripts/fdo_toolkit.sh

```diff
@@ -0,0 +1,335 @@
+#    # Install docker-compose from its github repo, because that is the only way to get a recent enough version
+#    curl --progress-bar -L "https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose ## TODO link not working, See with Davis if we should keep this commented
+#    chk $? 'downloading docker-compose'
+#    chmod +x /usr/local/bin/docker-compose
+#    chk $? 'making docker-compose executable'
+#    ln -s /usr/local/bin/docker-compose /usr/bin/docker-compose
+#    chk $? 'linking docker-compose to /usr/bin'
+fi
+
+
+echo "Upgraded and Installed all FDO dependencies"
+}
+
+
+update_scripts() {
+if [ ! -e "$1" ]; then
+  echo "Error: $1 does not exist" >&2
+  exit 1
+fi
+cd $1
+sed -i 's/#subjectAltName = @alt_names/subjectAltName = @alt_names/g' web-server.conf
+sed -i 's/#subjectAltName/subjectAltName/g' web-server.conf
+head -n -8 web-server.conf > temp.txt ; mv temp.txt web-server.conf
+
+echo "[ alt_names ]">> web-server.conf
+echo "DNS.1 = $ipaddress" >> web-server.conf
+echo "DNS.2 = localhost" >> web-server.conf
+echo "DNS.3 = host.docker.internal" >> web-server.conf
+machine_dns=`cat /etc/hostname`
+echo "DNS.4 = ${machine_dns}" >> web-server.conf
+echo "IP.1 = $ipaddress" >> web-server.conf
+echo "IP.2 = 127.0.0.1" >> web-server.conf
+echo "IP.3 = 172.17.0.1" >> web-server.conf
+
+}
+
+generate_keys() {
+if [ ! -e "$1" ]; then
+  echo "Error: $1 does not exist" >&2
+  exit 1
+fi
+cd $1
+bash demo_ca.sh
+bash web_csr_req.sh
+bash user_csr_req.sh
+bash keys_gen.sh
+chmod +r secrets/server-key.pem
+cp -r secrets service.env ../manufacturer/
+cp -r secrets service.env ../rv/
+cp -r secrets service.env ../owner/
+cp -r secrets service.env ../aio/
+cp -r secrets ../db/
+}
+
+start_db() {
+if [ ! -e "$1" ]; then
+  echo "Error: $1 does not exist" >&2
+  exit 1
+fi
+cd $1/db
+if [ ! "$( docker container inspect -f '{{.State.Status}}' db_fdo-db_1 )" == "running" ]; then
+  sed -i 's+innodb_change_buffer_max_size = 25+#innodb_change_buffer_max_size = 25+g' custom/config-file.cnf
+  docker-compose up --build -d
+    if [ ! "$( docker container inspect -f '{{.State.Status}}' db_fdo-db_1 )" == "running" ]; then
+       echo "Failed to start DB container" >&2
+       exit 1
+    fi
+fi
+}
+
+start_aio() {
+if [ ! -e "$1" ]; then
+  echo "Error: $1 does not exist" >&2
+  exit 1
+fi
+cd $1/aio
+sed -i 's/  #- org.fidoalliance.fdo.protocol.UntrustedRendezvousAcceptFunction/  - org.fidoalliance.fdo.protocol.UntrustedRendezvousAcceptFunction/g' service.yml
+sed -i 's/  - org.fidoalliance.fdo.protocol.db.TrustedRendezvousAcceptFunction/  #- org.fidoalliance.fdo.protocol.db.TrustedRendezvousAcceptFunction/g' service.yml
+if [ ! "$( docker container inspect -f '{{.State.Status}}' pri-fdo-aio )" == "running" ]; then
+  sed -i 's+innodb_change_buffer_max_size = 25+#innodb_change_buffer_max_size = 25+g' custom/config-file.cnf
```

Comment:
This sed operation is not required, as its not part of our current config-file.cnf.This was removed as part of DB Upgradation